common/src/test/java/com/redhat/cloud/notifications/XRhIdentityUtilsTest.java

```diff
@@ -0,0 +1,64 @@
+package com.redhat.cloud.notifications;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class XRhIdentityUtilsTest {
+
+    /**
+     * Tests that when generating an encoded x-rh-identity content with both
+     * an account number and an org id, the content is encoded as expected.
+     */
+    @Test
+    void generateEncodedXRhIdentity() {
+        final String got = XRhIdentityUtils.generateEncodedXRhIdentity("ebs-account-number", "organization-id");
+
+        final String expectedValue = "eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6ImVicy1hY2NvdW50LW51bWJlciIsIm9yZ19pZCI6Im9yZ2FuaXphdGlvbi1pZCJ9fQ==";
+        Assertions.assertEquals(expectedValue, got, "the x-rh-identity contents were not correctly base64 encoded");
```

Comment:
For all tests in this class, you could also check that the result of `Base64Utils.decode(got)` contains the expected JSON.

Reply:
I've added the decoding testing as well in the test!